integration_tests/tests/s3.rs

```diff
@@ -582,25 +816,20 @@ fn test_delete_object(client: &TestClient, bucket: &str, filename: &str) {
     println!("{:#?}", result);
 }
 
-fn test_list_buckets(client: &TestClient) {
-    let result = client.list_buckets().sync().expect("Couldn't list buckets");
-    println!("\nbuckets available: {:#?}", result);
-}
-
-fn list_items_in_bucket(client: &TestClient, bucket: &str) {
-    let list_obj_req = ListObjectsV2Request {
-        bucket: bucket.to_owned(),
-        start_after: Some("foo".to_owned()),
-        ..Default::default()
-    };
-    let result = client
-        .list_objects_v2(list_obj_req)
-        .sync()
-        .expect("Couldn't list items in bucket (v2)");
-    println!("Items in bucket: {:#?}", result);
-}
-
-fn list_items_in_bucket_paged_v1(client: &TestClient, bucket: &str) {
+//fn list_items_in_bucket(client: &S3Client, bucket: &str) {
```

Comment:
Looks like this commented out code is covered in a test below. I think we can remove it, unless I'm missing something.

Reply:
You're right, must have missed that.